secator/query/json.py

```
@@ -194,3 +194,44 @@ def _execute_update(self, query: dict, update: dict) -> int:
+						'path': str(child),
+					})
+		return workspaces
+
+	def get_workspace(self, workspace_id: str):
+		"""Get workspace info from local filesystem."""
+		workspace_path = self.reports_dir / sanitize_folder_name(workspace_id)
+		if workspace_path.exists():
+			return {'workspace_id': workspace_id, 'workspace_name': workspace_id, 'path': str(workspace_path)}
+		return None
+
+	def list_runners(self, workspace_id: str = None, runner_type: str = None):
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)